msgpack/documentation/Readme.md

```diff
@@ -0,0 +1,45 @@
+# msgpack module documentation
+
+A module for **msgpack**, an efficient binary serialization format like JSON, but smaller and faster .
+
+## **msgpack**
+
+
++ **msgpack:sbuffer()** : construct the buffer that must containes packed data ;
++ **msgpack:objectHandle()** : construct a msgpack::object_handle. It must manage and return msgpack::object ;
++ **msgpack:object(msgpack_object_handle)** : construct the msgpack::object. It must containe the unpacked object ;
++ **msgpack:pack(sbuf object)** : data packing ;
++ **msgpack:unpack()** : for data unpacking ;
++ **msgpack:convert()** : convert msgpack::object to ark usable object .
```

Comment:
Could you use ArkScript notation for function calls instead of C-notation? (using `(name args)` instead of `name(args)`)

Also, it's "contain", not "containes" nor "containe"

Reply:
ok